src/wlapplication_messages.cc

```diff
@@ -19,177 +19,236 @@
 
 #include "wlapplication_messages.h"
 
+#include <algorithm>
 #include <iostream>
+#include <regex>
 
 #include <boost/format.hpp>
 
 #include "base/i18n.h"
 
 using std::endl;
 
+static size_t kIndent = 23;
+
+#ifndef _WIN32
+static std::string kDefaultHomedir = "~/.widelands";
+#else
+static std::string kDefaultHomedir = "%USERPROFILE%\\.widelands";
+#endif
+
+/// Command line help
+/// Title: unindented text in the line above
+/// Key: the actual parameter
+/// Hint: text after =
+/// Help: Full text help
+/// Verbose: Filter some config options (--help vs. --help-all)
+/// Note that the _() markup is only for extraction. Since no textdomain is available in
+/// static context these should be marked again on usage.
+static std::vector<Parameter> parameters = {
+   {_("\nUsage:"), _("\twidelands <option0>=<value0> ... <optionN>=<valueN>"), "--", "", false},
```

Comment:
The translators will report all leading or trailing newlines,whitespaces,tabs in translatable strings as string issues